vignettes/download_jira_data.Rmd

```diff
@@ -42,6 +42,8 @@ issue_tracker_domain <- conf[["issue_tracker"]][["jira"]][["domain"]]
 issue_tracker_project_key <- conf[["issue_tracker"]][["jira"]][["project_key"]]
 save_path_issue_tracker_issues <- conf[["issue_tracker"]][["jira"]][["issues"]]
 save_path_issue_tracker_issue_comments <- conf[["issue_tracker"]][["jira"]][["issue_comments"]]
+
+save_path_issue_tracker_issues <- "../../rawdata/issue_tracker/geronimo/issues"
```

Comment:
See my comment on these variable names on the downloader. Also not sure why ths is being hardcoded instead of being read from the config file on line 43.

Reply:
Ok (as for why I hardcoded the variable name, I did that for the demo we did. I just forgot to get rid of the changes I made.)